### ISSUE TITLE
Cannot run tests with ts-node

### ISSUE THREAD

**Issue by kirilloid** [wallabyjs/public#1784]
### Issue description or question

Wallaby doesn't run any tests.

I think the problem is with my configuration, but I have found neither matching example nor a better place to ask. All source code is in TypeScript, regular build is performed by `webpack`, but I run tests w/o transpiling—using `ts-node`.

Test console shows: `​​​​​0 failing tests, 0 passing​​​​​`

Main console shows:
```​​
[Error] Error: spawn ts-node ENOENT​​
​​[Error]     at _errnoException (util.js:1003:13)​​
​​[Error]     at Process.ChildProcess._handle.onexit (internal/child_process.js:201:19)​​
​​[Error]     at onErrorNT (internal/child_process.js:389:16)​​
​​[Error]     at process._tickCallback (internal/process/next_tick.js:152:19)​​
```

I've tried set `runner` to `./node_modules/.bin/ts-node`, but it didn't work as well.

All code is available here: https://github.com/kirilloid/travian

### Wallaby.js configuration file

```javascript
module.exports = function(wallaby) {
    return {
        files: [
            'src/**/*.ts',
        ],
        tests: [
            'src/**/*.spec.ts'
        ],
        env: {
            type: 'node',
            runner: 'ts-node'
        }
    };
};
```

### Code editor or IDE name and version

Visual Studio Code v1.25.1

### OS name and version

OSX 10.13.6

**Comment by ArtemGovorov** [comment]
Wallaby compiles TypeScript before running your tests, so you don't need `ts-node`, try this:

```diff
module.exports = function(wallaby) {
    return {
        files: [
            'src/**/*.ts',
        ],
        tests: [
            'src/**/*.spec.ts'
        ],
        env: {
            type: 'node',
-           runner: 'ts-node'
        }
    };
};
```

**Comment by kirilloid** [comment]
Thanks for the prompt answer. I faced several other problems, the last of which I cannot resolve.

- I removed `runner` and it went further
- console started saying: "tests are also declared as files"—I've added negation pattern for files: `!src/**/*.spec.ts`
- now it displays whole log for `npm run test` in wallaby console and `0 failing tests, 0 passing​​​​​` in tests console even though there is a bunch of tests and one is failing.
